l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -45,17 +52,32 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
     /// @dev used to accept the admin role
     address private pendingAdmin;
 
+    IMessageRoot public override messageRoot;
+
     /// @notice Mapping from chain id to encoding of the base token used for deposits / withdrawals
     mapping(uint256 chainId => bytes32 baseTokenAssetId) public baseTokenAssetId;
 
+    ISTMDeploymentTracker public stmDeployer;
```

Comment:
interface vs storage variable name is confusing, shall it be the stmDeploymentTracker then?